### PR TITLE
[11.0][IMP] portal_partner_isr: add name and address to URL

### PR DESCRIPTION
**Pull request by mclaeysb**
Improvement of #76

### CODE REVIEW

**Comment by mclaeysb** [comment]
FYI: this branch is waiting for client confirmation before it can be merged.

**Review by remytms — APPROVED** [review]

**Comment by carmenbianca** [comment]
@remytms the repo template is not at all implemented on v11.0. I satisfied pre-commit in #270, but I'm not sure I want to go down the rabbit hole of fixing all the tests/Odoo dependencies.

It's OK to just merge manually.

**Review by carmenbianca — APPROVED** [review]

**Pull request merged**
